Add manage section to TOC terms (#214)
grafana/writers-toolkit: docs/sources/writing-guide/documentation-structure/index.md

| _Query data_ | [TraceQL](/docs/tempo/latest/traceql/) | Query languages, query tools, and examples. |
| _Visualize data_ | [Dashboards](/docs/grafana/latest/dashboards/) | Dashboard concepts and procedures. Link to the definitive source of dashboard documentation, rather than duplicating the information here. |
| _Alert_ | [Alerting rules](/docs/loki/latest/rules/) | This topic level is used for pages that discuss alerting features, like Grafana Loki's alerting rules. It provides a place for alerting content that is not specific to the Grafana Operations products. |

> | _Manage [product]_ | [Manage users and teams with Grafana Oncall](/docs/grafana-cloud/oncall/configure-user-settings/) | Information about managing a Grafana Labs product. |

Author: ```suggestion
| _Manage [product]_ | [Manage users and teams with Grafana Oncall](/docs/grafana-cloud/oncall/configure-user-settings/) | Information about managing a Grafana Labs product. Managing refers to maintaining what you have set up, for example,  viewing, editing, iterating on what you have created. |
```

> | _Manage [product]_ | [Manage users and teams with Grafana Oncall](/docs/grafana-cloud/oncall/configure-user-settings/) | Information about managing a Grafana Labs product. For example, content that helps you view, edit, and iterate on what you have created. |

Author: ```suggestion
| _Manage [product]_ | [Manage users and teams with Grafana Oncall](/docs/grafana-cloud/oncall/configure-user-settings/) | Information about managing a Grafana Labs product. For example, content in this topic helps you view, edit, and iterate on the Grafana product you installed. |
```
